kubernetes/cmsweb/services/crabcache.yaml

```diff
@@ -73,8 +73,8 @@ spec:
           mountPath: /etc/grid-security/hostcert.pem
           readOnly: true 
         # TMP: use cinder storage for time being
-        - name: crabcache-storage
-          mountPath: /data/srv/state/crabcache/files
+#PROD#        - name: crabcache-storage
```

Comment:
Muhammad, this is wrong indentation, see line 78 how it is done. The position of `- name` should remain the same as before. The `#PROD#` are replaced with 6 spaces by scripts/deploy.sh see https://github.com/dmwm/CMSKubernetes/blob/master/kubernetes/cmsweb/scripts/deploy.sh#L505